### PR TITLE
Added pear posting modal framework, reorganized file structure slightly

### PR DESCRIPTION
**Pull request by Zach-Kauffman**
want to merge so we can get this hooked up with the backend asap

### CODE REVIEW

File: src/public/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const insertNewPear = (pimage, ptitle, pauthor) => {
+    var newPear = document.querySelector("main.pear-container");
+    var pearContent = Handlebars.templates.pear;
+    var pearHTML = pearContent({image: pimage, title: ptitle, author: pauthor});
```
Comment (kalcicd) [inline src/public/index.js]: these should be const but nbd

File: src/server.js [hunk 2]
```diff
@@ -69,9 +69,9 @@ function close(req) {
 
 /**
  * Capture the port configuration for the server. We use the PORT environment
- * variable's value, but if it is not set, we will default to port 3000.
+ * variable's value, but if it is not set, we will default to port 6969.
  */
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 6969;
```
Comment (kalcicd) [inline src/server.js:74]: nice

File: src/views/partials/pear.hbs [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    <div class="pear-content">
+        {{image}}
+    <p class="pear-title">
+        {{title}}
+    </p>
+    <p class="pear-poster">
+        <a href="#">{{author}}</a>
+    </p>
+    </div>
+</article>
```
Comment (kalcicd) [inline src/views/partials/pear.hbs]: newline at end of file

File: src/views/layouts/main.hbs [hunk 4]
```diff
@@ -4,9 +4,11 @@
 <head>
 
     <meta charset="utf-8">
-    <title>Node.js Template</title>
+    <title>PearScale</title>
 
-    <link rel="stylesheet" href="/src/public/style.css" media="screen">
+    <link rel="stylesheet" href="style.css" media="screen">
+    <script src="index.js"></script>
+    <script src=
```
Comment (kalcicd) [inline src/views/layouts/main.hbs]: delete this fam

File: src/views/pearTemplate.hbs [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<main class = 'pear-container'>
+    {{> pear this}}
+</main>
```
Comment (kalcicd) [inline src/views/pearTemplate.hbs]: same as above

**Review by kalcicd — CHANGES_REQUESTED** [review]

File: src/views/pearTemplate.hbs (hunk 5 above)
Comment (Zach-Kauffman) [reply src/views/pearTemplate.hbs]: this was in the tweeter thing so i went ahead and added it, ill remove this file for now

File: src/public/index.js (hunk 1 above)
Comment (Zach-Kauffman) [reply src/public/index.js]: fixed

File: src/views/layouts/main.hbs (hunk 4 above)
Comment (Zach-Kauffman) [reply src/views/layouts/main.hbs]: fixed

**Pull request merged**